src/js/stores/ChronosStore.js

```diff
@@ -0,0 +1,117 @@
+import {EventEmitter} from 'events';
+
+import AppDispatcher from '../events/AppDispatcher';
+import ChronosActions  from '../events/ChronosActions';
+import {
+  CHRONOS_JOBS_CHANGE,
+  CHRONOS_JOBS_ERROR,
+  VISIBILITY_CHANGE
+} from '../constants/EventTypes';
+import Config from '../config/Config';
+import JobTree from '../structs/JobTree';
+import {
+  REQUEST_CHRONOS_JOBS_ERROR,
+  REQUEST_CHRONOS_JOBS_ONGOING,
+  REQUEST_CHRONOS_JOBS_SUCCESS,
+  SERVER_ACTION
+} from '../constants/ActionTypes';
+
+import VisibilityStore from './VisibilityStore';
+
+let requestInterval;
+
+function startPolling() {
+  if (!requestInterval) {
+    ChronosActions.fetchJobs();
+    requestInterval = global.setInterval(
+      ChronosActions.fetchJobs,
+      Config.getRefreshRate()
+    );
+  }
+}
+
+function stopPolling() {
+  if (requestInterval) {
+    global.clearInterval(requestInterval);
+    requestInterval = null;
+  }
+}
+
+function handleInactiveChange() {
+  let isInactive = VisibilityStore.get('isInactive');
+  if (isInactive) {
+    stopPolling();
+  }
+
+  if (!isInactive && ChronosStore.shouldPoll()) {
+    startPolling();
+  }
+}
+
+VisibilityStore.addChangeListener(VISIBILITY_CHANGE, handleInactiveChange);
+
+class ChronosStore extends EventEmitter {
+  constructor() {
+    super(...arguments);
+
+    this.data = {
+      jobTree: {id: '/', items: []}
+    };
+
+    // Handle app actions
+    this.dispatcherIndex = AppDispatcher.register(({source, action}) => {
```

Comment:
I'm not sure what you're pointing out with your example. Maybe if you could explain a little more.

Can we agree that there's code which has made it through review that later we discovered to be inconsistent. This is quite confusing to developers, specially if I'm going to create a new Store, which style do I use? We should manage our technical debt and not create code which is inconsistent.
So please, let us do one of two things:
* Keep this store as it is with these new styles and refactor the rest of the stores to match this style now.
* Let's change the styles of this store to match the rest of our stores and at a later point refactor all stores at once to this new style.